client/src/App.js

```diff
@@ -1,29 +1,34 @@
 import React, { Component } from 'react';
-
+import { Route } from 'react-router-dom';
 import SavedList from './Movies/SavedList';
 import MovieList from './Movies/MovieList';
-import Movie from './Movies/Movie';
+import Movie from './Movies/Movie'
 
 export default class App extends Component {
-  constructor() {
-    super();
+  constructor(){
+    super()
     this.state = {
       savedList: []
-    };
+    }
   }
 
-  addToSavedList = movie => {
+  addToSavedList = (movie) => {
+    console.log(this.state.savedList)
     const savedList = this.state.savedList;
     savedList.push(movie);
-    this.setState({ savedList });
-  };
+    this.setState({savedList});
+  }
 
-  render() {
+  render(){
     return (
       <div>
         <SavedList list={this.state.savedList} />
-        <div>Replace this Div with your Routes</div>
+        <Route exact path="/" component={MovieList} />
+        <Route path="/movies/:id" render={ (props) => {
+          return(<Movie {...props} addToSavedList={this.addToSavedList}/>)
+        }} />
```

Comment:
Nicely done using render props. Note: you can utilize implicit return function on arrow function for cleaner code:
```
 props => <Movie {...props} addToSavedList={this.addToSavedList} />
```